solutions/cses-1640.mdx

```diff
@@ -141,3 +123,165 @@ print("IMPOSSIBLE")
 </PySection>
 
 </LanguageSection>
+
+## Solution 2 - Two Pointers
+
+By keeping two pointers, one at each end of the list, we can find the two indices in $\mathcal{O}(N)$ time.
```

Comment:
Maybe expand on this explanation a bit more? You can say something like if the current sum of the values at each of the pointers is too large, decrement the right pointer and if the sum is too small, increment the left pointer.